shell/pages/c/_cluster/settings/links.vue

```diff
@@ -0,0 +1,203 @@
+<script>
+import Loading from '@shell/components/Loading';
+import AsyncButton from '@shell/components/AsyncButton';
+import { Banner } from '@components/Banner';
+import { MANAGEMENT } from '@shell/config/types';
+import { fetchOrCreateSetting, SETTING } from '@shell/config/settings';
+import { _EDIT, _VIEW } from '@shell/config/query-params';
+import KeyValue from '@shell/components/form/KeyValue';
+import { allHash } from '@shell/utils/promise';
+
+const DEFAULT_CUSTOM_LINKS = [
+  {
+    order: 1,
+    key:   'Docs',
+    value: 'https://rancher.com/docs/rancher/v2.6/en'
+  },
+  {
+    order: 2,
+    key:   'Forums',
+    value: 'https://forums.rancher.com/'
+
+  },
+  {
+    order: 3,
+    key:   'Slack',
+    value: 'https://slack.rancher.io/'
+  },
+  {
+    order: 5,
+    key:   'Getting Started',
```

Comment:
as mentioned above, these shouldn't be en-us but a key that can be used to lookup the values in l10n files